internal/generator/vector/pipelines.go

```diff
@@ -62,6 +63,62 @@ if .log_type == "application" {
 `
 			vrls = append(vrls, parse)
 		}
+		if p.Schema == constants.OtelSchema && genhelper.IsOtelOutput(op){
+			schema := `
+					.timeUnixNano = to_unix_timestamp(to_timestamp!(del(.@timestamp)))
+					.severityText = del(.level)
+			  
+					# Convert syslog severity to number, default to 9 (unknown)
```

Comment:
Do we already do this for syslog outputs?

Reply:
VRL has functions to convert syslog levels/text to their appropriate counterparts. 
See : 
 - [Syslog Severity](https://vector.dev/docs/reference/vrl/functions/#to_syslog_severity)
 - [Syslog Level](https://vector.dev/docs/reference/vrl/functions/#to_syslog_level)